src/components/SpiceList.js

```diff
@@ -21,10 +38,10 @@ class SpiceList extends React.Component {
           <div className="filters">
             <div>
               <label>Search: </label>
-              <input type="text" placeholder="Search By Tasting Notes..." />
+              <input onChange={this.handleSearch} type="text" placeholder="Search By Tasting Notes..." />
```

Comment:
Well done getting the `onChange` handler set up here to update state! When you're working with inputs, React also recommends making them [controlled](https://reactjs.org/docs/forms.html#controlled-components) by connecting the `value` attribute of the input to a key in state:

```jsx
<input onChange={this.handleSearch} value={this.state.search} type="text" placeholder="Search By Tasting Notes..." />
```